code/controlconfig/controlsconfig.cpp

```diff
@@ -482,7 +483,12 @@ int joy_get_scaled_reading(int raw)
 
 	raw -= JOY_AXIS_CENTER;
 
-	dead_zone = (JOY_AXIS_MAX - JOY_AXIS_MIN) * Joy_dead_zone_size / 100;
+	if (Cmdline_deadzone >= 0)	{
+		dead_zone = (JOY_AXIS_MAX - JOY_AXIS_MIN) * Cmdline_deadzone / 200; //Making this div by 200, the max deadzone value in cmdline is 100, which is more intuitive
+	}
+	else{
```

Comment:
Nitpick comment, but have the `} else {` on the same line with a space after the else is more consistent with other code formatting.

Reply:
Please be consistent with the style.

`} else {`

Reply:
Maybe add comment above or close by explaining what the max deadzone size is, since its not immediately apparent just looking at this line here.

Original code has `Joy_dead_zone_size` to be 5 * the number of dots the slider has (0 to 10), thus has the range of 0 to 50.  Thus, `Joy_dead_zone_size / 100` has the range of 0f to 0.5f

Reply:
My intent was to allow the user to go beyond the original maximum as well. The way the PR is right now, the user could have the deadzone be 99% of the range, or more (making the joystick unresponsive)